test/html/multiple-imports.html

```diff
@@ -7,5 +7,6 @@
     Code distributed by Google as part of the polymer project is also
     subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
+<link rel="import" href="multiple-imports.html">
```

Comment:
Fair point.  The test needs to be more intention revealing.  Essentially, bundles with files that are not a part of the original transitive dependency chain will suffer from this problem, since imports are appended in arbitrary order to the hidden div.

In the case of running in polymer-build, what I saw happen was that the common-dependency.html was injected into the shell's bundle by the bundle strategy, because that's what it should do, and shell.html's hidden div wound up with a `<link rel=import href=shell.html><link rel=import href=common-dependency.html>` before inlining, because appendHtmlImports adds references to all files, and then the common-dependency.html wasn't never inlined because the condition that we've "already visited the importBundleUrl of shell.html" short-circuited subsequent imports.